docs/pages/installation/aks.md

```diff
@@ -61,14 +63,14 @@ kubectl cluster-info
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
  $ kubectl create ns meshery
- $ helm repo add meshery https://meshery.io/charts/
- $ helm install meshery --namespace meshery
+ $ helm repo add meshery https://meshery.io/charts
+ $ helm install meshery meshery/meshery --namespace meshery

```

Comment:
Where is that path `meshery/meshery` ?

Reply:
If we add the meshery Helm chart's repo, we can install meshery directly right? Why we need to add the path of the chart?

Reply:
@Aisuko the first `meshery` of `meshery/meshery` refers to the added repo name, and the second `meshery` refers to the chart name under the repo.

The commands look good to me, except I'd prefer to combine `kubectl create ns meshery` and `helm install` into one command by adding the `--create-namespace` flag, e.g:
`helm install meshery meshery/meshery --namespace meshery --create-namespace`

Reply:
Ok, thanks @panyuenlau. For me, I'd like to suggest add the comment to the command above. And `--create-namespace` is a great parameter for deploying the Helm chart without create namesapce manually.